docs/root/faq/zone_aware_routing.rst

```diff
@@ -18,22 +18,18 @@ These are the requirements:
 
   Only essential parts are listed in the configuration below for the cluster manager.
 
-.. code-block:: json
-
-  {
-    "sds": "{...}",
-    "local_cluster_name": "cluster_a",
-    "clusters": [
-      {
-        "name": "cluster_a",
-        "type": "sds",
-      },
-      {
-        "name": "cluster_b",
-        "type": "sds"
-      }
-    ]
-  }
+.. code-block:: yaml
+
+  cluster_manager:
+    local_cluster_name: cluster_a
+  static_resources:
+    clusters:
+    - name: cluster_a
+      type: EDS
+      eds_cluster_config: "{...}"
```

Comment:
Rather than use `"{...}"` for YAML, I would use `...`, since we don't have a string situation here.